rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -183,16 +203,20 @@ def _decode_fpis_claim(
             counterparty = CPT_FRAX
 
         for event in context.decoded_events:
-            if event.event_type == HistoryEventType.RECEIVE and event.location_label == user_address and amount == event.balance.amount and self.fpis == event.asset:  # noqa: E501
-                event.event_type = HistoryEventType.RECEIVE
-                event.event_subtype = HistoryEventSubType.AIRDROP
-                event.counterparty = counterparty
-                try:
-                    crypto_asset = event.asset.resolve_to_crypto_asset()
-                except (UnknownAsset, WrongAssetType):
-                    self.notify_user(event=event, counterparty=counterparty)
-                    continue
-                event.notes = f'Claim {amount} {crypto_asset.symbol} {note_location}'
+            notes = event.notes
+            try:
+                notes = f'Claim {amount} {event.asset.resolve_to_crypto_asset().symbol} {note_location}'  # noqa: E501
+            except (UnknownAsset, WrongAssetType):
+                self.notify_user(event=event, counterparty=counterparty)
+
+            if make_claim(
+                event,
```

Comment:
can you use the kwargs in all the arguments?

```suggestion
                event=event,
```